documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc

```diff
@@ -44,11 +44,13 @@ This is available via device-tree in `/proc/device-tree/chosen/rpi-boardrev-ext`
 * Raspberry Pi 400
  ** Bits 0-7: The default keyboard country code used by https://github.com/raspberrypi-ui/piwiz[piwiz]
 
-36-43 - xref:raspberry-pi.adoc#industrial-use-of-the-raspberry-pi[customer OTP values] +
-45 - MPG2 decode key +
-46 - WVC1 decode key +
-47-55 - Reserved for signed-boot on Compute Module 4 +
-64/65 -- MAC address; if set, system will use this in preference to the automatically generated address based on the serial number  +
+36-43 -- xref:raspberry-pi.adoc#industrial-use-of-the-raspberry-pi[customer OTP values] +
+45 -- MPG2 decode key +
+46 -- WVC1 decode key +
+47-54 -- SHA256 of RSA public key for secure-boot +
+55    -- secure-boot flags (reserved for use by the bootloader) +
+56-63 -- 256bit device specific private key +
```

Comment:
Minor quibble, but IMHO 256bit should be 256-bit. And perhaps a hyphen between 'device' and 'specific' ?

Reply:
Not sure it matters. Would be better to publish now since the code is out there but feel free to edit directly